Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            PersonInfoAskerAndPrinter();
+            PersonInfoAskerAndPrinter();
+        }
+        private static void PersonInfoAskerAndPrinter()
+        {
+            string firstName = PromptString("What is your first name: ");
+            string lastName = PromptString("What is your last name: ");
+            int age = PromptInt("What is your age: ");
+            float weight = PromptFloat("what is your weight in Kg: ");
+            float height = PromptFloat("what is your height in cm: ");
+            Console.WriteLine($"{firstName} {lastName} is {age} years old, his weight is {weight} kg and his height is {height} cm. ");
+            float bmi = BmiCalculator(weight, height / 100);
+            Console.WriteLine($"Your BMI is {bmi}");
+        }
```

Comment:
Add a space between your function declarations to improve readability.

Reply:
ah yes i see and agree! thanks for the tip.